website_docs/getting-started.md

```diff
@@ -7,6 +7,8 @@ Welcome to the OpenTelemetry for Go getting started guide! This guide will walk
 
 Understand how a system is functioning when it is failing or having issues is critical to resolving those issues. One strategy to understand this is with tracing. This guide shows how the OpenTelemetry Go project can be used to trace an example application. You will start with an application that computes Fibonacci numbers for users, and from there you will add instrumentation to produce tracing telemetry with OpenTelemetry Go.
 
+For reference, a complete example of the code you will build can be found [here](https://github.com/open-telemetry/opentelemetry-go/tree/main/example/fib).
```

Comment:
Should this link to a particular tag or commit rather than `main`?  It's unlikely that this example will be removed, but who knows.

Reply:
It seems to me that link should always be as current as possible (hence main). How about a GitHub Action looking for broken links?

Reply:
There'a actually already one: https://github.com/open-telemetry/opentelemetry-go/blob/main/.github/workflows/markdown.yml#L36

Reply:
@Aneurysm9 thoughts ^?

Reply:
We'll see how it works.